libc/string/arc/memcpy.S

```diff
@@ -1,19 +1,15 @@
 /*
- * Copyright (C) 2013, 2014-2015, 2017 Synopsys, Inc. (www.synopsys.com)
+ * Copyright (C) 2013-2022 Synopsys, Inc. (www.synopsys.com)
```

Comment:
Copyright years must reflect only years when changes were really done. In that case it should be `2013, 2014-2015, 2017, 2022`.